Win2D.proj

```diff
@@ -233,7 +233,7 @@
     <_VsTest Condition="Exists('$(VS120COMNTOOLS)$(_VsTestPathUnderCommonTools)')">$(VS120COMNTOOLS)$(_VsTestPathUnderCommonTools)</_VsTest>
     <_VsTest Condition="Exists('$(VS140COMNTOOLS)$(_VsTestPathUnderCommonTools)')">$(VS140COMNTOOLS)$(_VsTestPathUnderCommonTools)</_VsTest>
     <_VsTest Condition="Exists('$(VS150COMNTOOLS)$(_VsTestPathUnderCommonTools)')">$(VS150COMNTOOLS)$(_VsTestPathUnderCommonTools)</_VsTest>
```

Comment:
This should be adding a new condition for VS**160**COMNTOOLS

Reply:
Nice catch!  I will test this out tomorrow.

Reply:
It executed all tests for me

Reply:
It actually required a few more tweaks since everything was updated. I'll push a new branch out of this one with a single new commit.

Reply:
Now it's running the tests the first time, but then it tries to build for different arch (x64 x Win32) and fails to run the tests because the appx with the same id already exists... It's not uninstalling or replacing the version with the new one, so I reduced from 4 bugs to 3. I've found this: https://github.com/Microsoft/vstest/issues/1520 Investigating...

Reply:
I see the same thing trying this locally.   The workaround mentioned in that vstest issue of manually uninstalling the package before attempting to run tests seems plausible?

Reply:
Not really, because the whole process is automated. This will never run on a vanilla build server without changing the build script. We need to either automatically try to uninstall the package before running the tests for each arch/config or change the generated package name for each arch/config. What do you think is the best solution?

Reply:
Yep, this would need to be automated in Win2D.proj to attempt an uninstall before running each test pass.

I think changing generated package names would still run into the same bug if more than one build was run back-to-back on the same machine?

Reply:
No, they would replace the old installation. The problem is that the IDs are different but the names are the same. It's not hard to remove it from PS:
get-appxpackage -name E2C40EB9-8CCE-48B1-9A5C-4C8CA4999631 | remove-appxpackage
I'll send the fix.

Reply:
I sent #721 that has all the commits here and the tests works.